script.kodi.windows.update/default.py

```diff
@@ -15,8 +15,8 @@
 #
 # -*- coding: utf-8 -*-
 
-import os, time, datetime, traceback, re, fnmatch, glob, threading
-import urllib, urllib2, socket, subprocess, shutil, sys
+import os, time, datetime, traceback, re, threading, json
+import urllib, urllib2, socket, subprocess, sys
```

Comment:
same here. Let me know if you want me to merge this or if I should wait for py compatibility